library/item/item_setting/src/main/java/org/quiltmc/qsl/itemsetting/api/package-info.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * <h2>The Quilt Item Setting sub API.</h2>
```

Comment:
what does sub api mean?

Reply:
Mostly that its a part of the larger item API. This should be changed as I moved this into its own module